src/github3/pulls.py

```diff
@@ -495,6 +547,18 @@ def review_comments(self, number=-1, etag=None):
         url = self._build_url('comments', base_url=self._api)
         return self._iter(int(number), url, ReviewComment, etag=etag)
 
+    def review_requests(self):
+        """Retrieve the review requests associated with this pull request.
+
+        :returns:
+            review requests associated with this pull request
+        :rtype:
+            :class:`~github3.pulls.ReviewRequests`
+        """
+        url = self._build_url('requested_reviewers', base_url=self._api)
+        json = self._json(self._get(url), 200)
+        return self._instance_or_null(ReviewRequests, json)
```

Comment:
This method doesn't use `_iter` because GitHub's API appears to deviate from the usual pattern of list endpoints: the response is not a list of objects, but an object containing two lists (see: https://developer.github.com/v3/pulls/review_requests/#list-review-requests). Because of this, I created a new class (`ReviewRequests`) that models this response. I confess I'm not 100% happy with this design...